schemas/latest/dev-workspace-template.json

```diff
@@ -81,7 +81,8 @@
               },
               "fieldsV1": {
                 "description": "FieldsV1 holds the first JSON version format as described in the \"FieldsV1\" type.",
-                "type": "Any"
+                "type": "object",
+                "additionalProperties": false
```

Comment:
Is it expected to have an `object` with no properties and additionalproperties to `false`? That seems strange because if I'm not mistaken it just prevents putting anything in the corresponding field. Maybe this should be managed as a special case (`maganedFields` is somehow a special case in itself...)?

Reply:
I'll take a look.

Reply:
Good catch!
It was tricky but I found a workaround https://github.com/devfile/api/pull/524/commits/20a45168c071c58fec1f049f1e1d4ae6629ecaa3

But I think additional work is needed out of the current PR scope since(the main branch state) we generate all metadata additionalProperties: false, like managedField https://github.com/devfile/api/blob/main/schemas/latest/dev-workspace-template.json#L100
But K8s Json Schema does not specify any and AFAIU it's supposed to be true by default https://kubernetesjsonschema.dev/v1.14.0/_definitions.json#/definitions/io.k8s.apimachinery.pkg.apis.meta.v1.ObjectMeta
![Screenshot_20210709_140215](https://user-images.githubusercontent.com/5887312/125068550-47539480-e0be-11eb-8495-ed274720a140.png)